### ISSUE TITLE
There is a question.

### ISSUE THREAD

**Issue by QiyuZhu1** [ffhibnese/CGNC_Targeted_Adversarial_Attacks#6]
Dear Author, thank you very much for sharing your insights. I have a question to ask you. 
         When running get_text_feature.py, I often encounter insufficient GPU memory. May I ask if the clip model you used is ViT-B/16? What GPU memory capacity do you typically use on your server when running get_text_feature.py? How long does it take to generate the text_feature.pth file?

**Comment by ffhibnese** [comment]
Yes, we adopt ViT-B/16 on 24G 3090 GPU. It solely consumes a couple of seconds to finish the generation of text_feature.pth.

**Comment by QiyuZhu1** [comment]
Right, thank you! I have one further query I'd like to ask. This method requires a considerable amount of training time. I note you employed the model generated in the ninth round. My current model, trained in the first round, produces adversarial samples with a very low attack rate. Is this normal? Will the attack rate improve significantly with further training? Could you elaborate on how the attack rate of the model changes after each training round?

**Comment by ffhibnese** [comment]
Sorry, since this work has been finished a long time agao, I'm not quite sure about the ASR curve. I suggest that you complete the training and observe how the ASR evolves.

**Comment by QiyuZhu1** [comment]
Thank you. May I also ask whether the two pre-trained models provided in your code repository were obtained by directly running `python train.py --train_dir $DATA_PATH/ImageNet/train --model_type incv3 --start_epoch 0 --epochs 10 --label_flag “N8”`? Or were they further processed after training via `python train.py --train_dir $DATA_PATH/ImageNet/train --model_type incv3 --start_epoch 0 --epochs 15 --label_flag “N8”`? --train_dir $DATA_DIR/ImageNet/train --model_type incv3 --start_epoch 10 --epochs 15 --label_flag “N8” --load_path $CKPT_DIR/incv3/model-9.pth --finetune --finetune_class 150?

**Comment by KongJiawei1231** [comment]
The provided pre-trained model corresponds to the checkpoint saved after the 10th training epoch.

**Comment by QiyuZhu1** [comment]
Alright, thank you very much for your explanation. I'd like to reproduce your training results for comparative experiments. Due to limited GPU memory, I set the batch size to 8 for training and used Res50 as the proxy model, keeping all other hyperparameters unchanged. The target attack rates obtained after the first three rounds were 11.88%, 13.59%, and 14.12%, respectively. The current loss is oscillating around 0.4, showing a significant deviation from your results. I believe this discrepancy stems from operational errors on my part. I am eager to consult with you regarding the potential causes of this discrepancy and how to improve my approach to achieve results comparable to your generator. Your guidance would be greatly appreciated.
